kss/builder/kss-assets/images/cr_logo.svg

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could we add this to `spritesheet.svg`, and do the same within Drupal so it's easily reusable?